force-app/main/default/triggers/HiringTeamTrigger.trigger

```diff
@@ -0,0 +1,5 @@
+trigger HiringTeamTrigger on Hiring_Team__c (before insert, after insert, before update, after update, before delete, after delete, after undelete) {
+    HiringTeamHandler handler = new HiringTeamHandler();
+
```

Comment:
fix spacing, I would not put a space between these two lines of code and I would put a space before and after for clean and easy to read code.